uploader.v2.json

```diff
@@ -3368,7 +3354,22 @@
             "glob": "/sys/class/scsi_host/host[0-9]*/eh_deadline",
             "symbolic_name": "scsi_eh_deadline",
             "pattern": []
-        }
+        },
+	{
+            "glob": "/var/log/ceph/()ceph.log*",
+            "pattern": [
+                "[WRN] slow request"
+            ],
+            "symbolic_name": "ceph_log"
+        },
+        {
+            "glob": "/var/log/ceph/()*ceph-osd.*\\.log$",
```

Comment:
Please change this entry to: `"glob": "/var/log/ceph/*ceph-osd.*.log",`  This spec does not match the spec in insights-core which is `glob_file(r"var/log/ceph/ceph-osd*.log"`.  These need to match.

Reply:
Done